src/AWS.Deploy.CLI/Commands/CommandFactory.cs

```diff
@@ -208,6 +217,7 @@ private Command BuildDeployCommand()
                         _toolInteractiveService.WriteErrorLine(string.Empty);
                         _toolInteractiveService.WriteErrorLine(e.Message);
                     }
+
```

Comment:
please remove all whitespace changes

Reply:
This change removes the redundant white spaces and therefore is intended